https://gn.googlesource.com/gn/+/c7b223bfb225ce87a72a244d016ffdfcf227fa5e%5E%21/
Ignore build warning -Werror=redundant-move

gcc-13 complains with:
error: redundant move in return statement [-Werror=redundant-move]

We cannot fix the code, because both old versions of gcc and the windows
toolchain fails to build.
index adb622a..232e536 100755
--- a/tools/gn/build/gen.py
+++ b/tools/gn/build/gen.py

@@ -472,6 +472,8 @@
     # flags not supported by gcc/g++.
     if cxx == 'clang++':
       cflags.extend(['-Wrange-loop-analysis', '-Wextra-semi-stmt'])
+    else:
+      cflags.append('-Wno-redundant-move')
 
     if platform.is_linux() or platform.is_mingw() or platform.is_msys():
       ldflags.append('-Wl,--as-needed')
